### PR TITLE
fix(github): fixing support for github enterprise urls

### PR DESCRIPTION
**Pull request by meldiner**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@bahmutov/parse-github-repo-url",
   "description": "Parse a GitHub URL for user/project@version",
-  "version": "0.0.0-semantic-release",
```
Comment (bahmutov) [inline package.json:4]: please put the semantic release back - this module is published by TravisCI automatically including determining the version

**Comment by bahmutov** [comment]
almost there, except for version

**Comment by meldiner** [comment]
fixed

**Comment by bahmutov** [comment]
thank you! Should be published shortly

**Comment by meldiner** [comment]
great, thanks!

**Pull request merged**